src/Features/Core/Portable/UseAutoProperty/AbstractUseAutoPropertyAnalyzer.cs

```diff
@@ -218,11 +216,19 @@ protected void AnalyzeProperty(SemanticModelAnalysisContext context, TPropertyDe
                 return;
             }
 
+            if (!CanConvert(property))
+            {
+                return;
+            }
+
             // Looks like a viable property/field to convert into an auto property.
             analysisResults.Add(new AnalysisResult(property, getterField, propertyDeclaration, fieldDeclaration, variableDeclarator,
                 property.ToDisplayString(SymbolDisplayFormat.FullyQualifiedFormat)));
         }
 
+        protected virtual bool CanConvert(IPropertySymbol property)
```

Comment:
*Convert* is misleading. Might be better to name it *CanBeFixed*

Reply:
Enh... 'Convert' is a term we use all over the code fixes for the same meaning: "can we convert the current form to final form?".

Reply:
You are right. *convert* is also used in the comment right below. But taking the method signature *CanConnvert(IPropertySymbol)* one can easily assume that this is asking about type conversion.

Reply:
I'm really ok with that.  As mentioned above, it's common terminology in our code-fixes.  I'd prefer to just keep using it :)